src/Adapter/OrderReturn/CommandHandler/BulkDeleteProductFromOrderReturnHandler.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
I can't add comment on empty file, but src/Adapter/OrderReturn/AbstractOrderReturnHandler.php is empty

Reply:
Deleted it